frontend-react/src/utils/DateTimeUtils.ts

```diff
@@ -56,3 +58,47 @@ export function formatDateWithoutSeconds(d: string) {
     });
     return newDate.replace(/,/, "");
 }
+
+/*
+ * format: "Mon, 7/25/2022"
+ * WARNING: Intl.DateTimeFormat() can be slow if called in a loop!
+ * Rewrote to just use Date to save cpu
+ * */
+export const dateShortFormat = (d: Date) => {
+    const dayOfWeek =
+        ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"][d.getDay()] || "";
```

Comment:
do we need this `""` fallback? if `d` is always a `Date` object, `d.getDay()` should always return something valid.

Reply:
This is copy+pasted original code, so not necessary to change for this scope of work.